<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>300</width>
      <height>315</height>
     </rect>
    </property>
    <property name="title">
     <string>Contacts list</string>
    </property>
    <widget class="QLineEdit" name="search_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>280</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="contactList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>280</width>
       <height>245</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>325</y>
      <width>580</width>
      <height>125</height>
     </rect>
    </property>
    <property name="title">
     <string>Message</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLineEdit" name="message_line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>360</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="send_button">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>90</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="attach_file_button">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add file</string>
     </property>
    </widget>
    <widget class="QListWidget" name="disp_msg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>560</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>115</height>
     </rect>
    </property>
    <property name="title">
     <string>User</string>
    </property>
    <widget class="QLabel" name="userName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>160</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;user_name&gt;</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Change state</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>270</width>
      <height>195</height>
     </rect>
    </property>
    <property name="title">
     <string>Find new contacts</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>140</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="search_new_button">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Add conatct</string>
     </property>
    </widget>
    <widget class="QListWidget" name="newContacts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>250</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>250</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogout"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_application"/>
    <addaction name="actionHelp"/>
    <addaction name="actionAuthors"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAuthors">
   <property name="text">
    <string>Authors</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout_application">
   <property name="text">
    <string>About application</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
